fdk.go

```diff
@@ -129,10 +132,85 @@ type HTTPContext interface {
 	RequestMethod() string
 }
 
+// TracingContext contains all configuration for a function invocated to
+// get the tracing context data.
+type TracingContext interface {
+	/**
+	 * Returns true if tracing is enabled for this function invocation
+	 * @return whether tracing is enabled
+	 */
+	IsTracingEnabled() bool
+
+	/**
+	 * Returns the user-friendly name of the application associated with the
+	 * function; shorthand for Context.getAppName()
+	 * @return the user-friendly name of the application associated with the
+	 * function
+	 */
+	GetAppName() string
+
```

Comment:
I agree with Isaac that that is not part of the TracingContext interface.
In addition to it a small nit: in golang is not common to use the Get for the Get methond, can you please get rid of the Get prefix in these methods definitions?